packages/e2e-tests/src/specs/dashboard/templates/useTemplate.js

```diff
@@ -20,6 +20,8 @@
 import { percySnapshot } from '@percy/puppeteer';
 import { visitDashboard } from '@web-stories-wp/e2e-test-utils';
 
+const percyCSS = `.dashboard-media-element { background-color: none; }`;
```

Comment:
This flakiness is not isolated to this test, but could occur anywhere media elements are displayed. So it's probably better to move this to `percy.config.yml`.